examples/models/llama2/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@
 cmake_minimum_required(VERSION 3.19)
 project(llama_runner)
 
+option(EXECUTORCH_REGISTER_OPTIMIZED_OPS "Build the optimized kernels" ON)
```

Comment:
Ok for now but can you check if user need to specify this option in two different places? One in executorch build and second here. I know by default is on but maybe just check if target `optimized_native_cpu_ops_lib` exist?

Reply:
I'll address in a follow-up.